plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -732,7 +732,7 @@ export default function TableChart<D extends DataRecord = DataRecord>(
       if (label.toLowerCase().includes('video_title_link'))
         columnObject.width = columnObject.width > 240 ? columnObject.width - 2 : 240;
       if (label.toLowerCase().includes('positive_sentiment_valence'))
-        columnObject.width = columnObject.width > 200 ? columnObject.width - 2 : 200;
```

Comment:
you also need to do changes for the following variable *marginLeftForHorizontalScroll*